draft-ietf-rats-corim.md

```diff
@@ -1107,6 +1110,16 @@ checks, and trust anchor checks.
 {::include cddl/crypto-key-type-choice.cddl}
 ~~~
 
+##### Integrity Registers {#sec-comid-integrity-registers}
+
+An Integrity register is a group of one or more named measurements.
+Each measurement has an unique identifier (either an unsigned integer or a string) and one or more associated digests.
+This type can be used to model the PCRs in a TPM or vTPM, in which case the identifier is the register index, as well as other vendor-specific "Integrity Registers".
```

Comment:
```suggestion
Integrity registers can be used to model the PCRs in a TPM or vTPM ( in which case the identifier is the register index), or vendor-specific "Integrity Registers".
```

Reply:
reworded in [0daa574](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/189/commits/0daa57453d485e1ad42dd2d338868d0b3d389fcd)